### PR TITLE
[WIP] Sundry changes to work with omnia-linux-anvil image

### PR DESCRIPTION
**Pull request by mpharrigan**

### CODE REVIEW

**Comment by jchodera** [comment]
Thanks again for the PR, @mpharrigan! I'll pick this up again after getting the openmm core 21 v0.0.18 fix out.

**Comment by mpharrigan** [comment]
I'm going to have to do some serious rebasing here, as this included some fixes for conda build > 2.0.1, which have been fixed (??) elsewhere

**Comment by mpharrigan** [comment]
rebased

**Comment by jchodera** [comment]
> I'm going to have to do some serious rebasing here, as this included some fixes for conda build > 2.0.1, which have been fixed (??) elsewhere

Shoot, sorry about that---I hadn't realized you had fixed some of those issues!

**Comment by jchodera** [comment]
I'll be able to get back to this immediately after the core21 0.0.18 issue. Sorry about the delay.

**Comment by mpharrigan** [comment]
> Shoot, sorry about that---I hadn't realized you had fixed some of those issues!

No problem, was painless

> I'll be able to get back to this immediately after the core21 0.0.18 issue. Sorry about the delay.

No problem, still WIP

File: devtools/docker-build.sh [hunk 1]
```diff
@@ -1,15 +1,8 @@
...
 set -e
-# Activate Holy Build Box environment.
-source /hbb_exe/activate
-
-# Disable PYTHONPATH
-unset PYTHONPATH
-
 set -x
-curl -s -O https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh
-bash Miniconda3-latest-Linux-x86_64.sh -b -p /anaconda
```
Comment (jchodera) [inline devtools/docker-build.sh]: Why remove the miniconda install? This is causing the build to fail.

You probably want to install a fresh Miniconda each time, as [we do now](https://github.com/omnia-md/conda-recipes/blob/master/devtools/docker-build.sh#L9-L14).

You'll also want to add the `conda-forge` channel, right?

Comment (mpharrigan) [inline devtools/docker-build.sh]: the linux anvil image includes miniconda

right now i explicitly remove the conda-forge channel or else it pulls all the conda-forge packages for python / conda / conda build

we can add it in later (if needed) making sure to give it less priority than defaults

Comment (jchodera) [inline devtools/docker-build.sh]: > the linux anvil image includes miniconda

How do they ensure that miniconda is curent? Or do they avoid doing that?

> right now i explicitly remove the conda-forge channel or else it pulls all the conda-forge packages for python / conda / conda build

Ah, OK.

Do we need to activate the conda version in the anvil, or put it in PATH?

**Comment by jchodera** [comment]
Heads up that @Lnaden and I are going to finally dive in and try to finish the CentOS 6 upgrade next week.

**Comment by Lnaden** [comment]
I have a new PR open which should supersede this one. It was easier to just do a new PR on the main repo here instead of a fork due to all the changes from the other files which have changed since this was open. When we merge the #761, this should auto-close.

**Pull request closed without merging**